src/layouts/Layout.jsx

```diff
@@ -30,31 +30,29 @@ class Layout extends React.PureComponent {
 
     return (
       <div>
-        <Grid fluid={ true }>
-          <Row>
-            <Navbar className='navBar'>
-              <Col md={ 3 } xs={ 12 }>
-                <Link
-                  className='link'
-                  to={ '/articles' }
-                  >
-                  <Image
-                    alt='freeCodeCamp logo'
-                    responsive={ true }
-                    src={
-                      'https://raw.githubusercontent.com/' +
-                      'freeCodeCamp/assets/master/assets/' +
-                      'logos/FCC-logo-white.png'
-                      }
-                  />
-                </Link>
-              </Col>
-              <Col md={ 9 } xs={ 12 }>
-                <SearchBar />
-              </Col>
-            </Navbar>
-          </Row>
-        </Grid>
+        <nav className='navbar navBar'>
+          <Grid className='navContainer'>
+            <Col className='hidden-xs' sm={ 3 }>
```

Comment:
I'm not sure about not showing the logo on smaller devices

Reply:
Absolutely agree. This is definitely not ideal, and this is what I meant about the responsiveness still not being great.

The reason I went with this is that there isn't enough space to show both the logo and the search bar. devdocs.io solved this by not showing a logo. freeCodeCamp.org solved this by only showing the logo and a menu button.

Ideally, we do something similar to the devdocs solution. Have the searchbar right there, have a button for the nav. There still wouldn't be space for a logo. My idea was to go with this for now to make it usable, then make it look better in a PR that fixes the navigation on smaller screens.

Reply:
@QuincyLarson hiding the logo on mobile?

Reply:
From @QuincyLarson 
> For the guide, we want the logo to be visible because one of our goals is to raise awareness of our community to people arriving from Google searches.

Reply:
@Bouncey I absolutely agree, but I don't know how it would be possible to show both the search bar and the logo on a small screen. How about putting the glyph version in place of the full logo? https://freecodecamp.github.io/design-style-guide/#logo
It's not the perfect branding because it doesn't show the name, but it might be the best compromise.

Reply:
@Bouncey perhaps on smaller breakpoints we would need to show the glyph. I imagine even an 80px wide search bar would suffice though. We have no other navigation.